en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml

```diff
@@ -3672,308 +3729,387 @@ paths:
+        This operation can be used to publish an API to a list of external stores.
+      operationId: publishAPIToExternalStores
       parameters:
         - $ref: '#/components/parameters/apiId'
+        - name: externalStoreIds
+          in: query
+          description: External Store Ids of stores which the API needs to be published
+            or updated.
+          schema:
+            type: string
         - $ref: '#/components/parameters/If-Match'
-      requestBody:
-        content:
-          multipart/form-data:
-            schema:
-              properties:
-                file:
-                  type: string
-                  description: WSDL file or archive to upload
-                  format: binary
-                url:
-                  type: string
-                  description: WSDL Definition url
       responses:
         200:
           description: |
             OK.
-            Successful response with updated WSDL definition
+            API was successfully published to all the selected external stores.
           headers:
             ETag:
               description: |
-                Entity Tag of the response resource. Used by caches, or in conditional requests (Will be supported in future).
+                Entity Tag of the blocked subscription.
+                Used by caches, or in conditional requests (Will be supported in future).
               schema:
                 type: string
             Last-Modified:
               description: |
-                Date and time the resource has been modifed the last time.
+                Date and time the subscription has been blocked.
                 Used by caches, or in conditional requests (Will be supported in future).
               schema:
                 type: string
-            Location:
-              description: |
-                The URL of the newly created resource.
+          content:
+            application/json:
               schema:
-                type: string
-            Content-Type:
-              description: |
-                The content type of the body.
+                $ref: '#/components/schemas/APIExternalStoreList'
+        404:
+          $ref: '#/components/responses/NotFound'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+      security:
+        - OAuth2Security:
+            - apim:api_publish
+            - apim:api_manage
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -X POST -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+          "https://127.0.0.1:9443/api/am/publisher/v3/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/publish-to-external-stores?externalStoreId=Store123#"'
+
+  /apis/{apiId}/generate-key:
+    post:
+      tags:
+        - APIs
+      summary: Generate internal API Key to invoke APIS.
+      description: |
+        This operation can be used to generate internal api key which used to invoke API.
```

Comment:
"API"

Kindly check in the other descriptions as well.

Reply:
Fixed